### ISSUE TITLE
Incorrect shell code in opal/mca/btl/uct/configure.m4

### ISSUE THREAD

**Issue by akesandgren** [open-mpi/ompi#6173]
Thank you for taking the time to submit an issue!

## Background information
opal/mca/btl/uct/configure.m4:
MCA_opal_btl_uct_CONFIG has a bug in setting btl_uct_LIBS

```
    AS_IF([test "$btl_uct_happy" = "yes"],
          [$1
           btl_uct_LIBS = "$btl_uct_LIBS -luct"
          ],
          [$2])
```
there should be no " " around the "=" when setting btl_uct_LIBS

### What version of Open MPI are you using? (e.g., v1.10.3, v2.1.0, git branch name and hash, etc.)
Master branch

**Comment by jsquyres** [comment]
I think you're referring to:

https://github.com/open-mpi/ompi/blob/c28ba954b9b81c55e9a6bc6cca44c2fa7b54fa7d/opal/mca/btl/uct/configure.m4#L52

Right?

If so, I think it should still have the quotes, but no extra spaces around ` = `, i.e.:

```sh
btl_uct_LIBS="$btl_uct_LIBS -luct"
```

Right?

**Comment by akesandgren** [comment]
exactly.

**Comment by hjelmn** [comment]
Easy enough to fix. Guess the shells I use are a little more permissive since this *seems* to be working on my system.

**Comment by jsquyres** [comment]
@akesandgren Hah!  I just realized I mis-read your initial report, and thought you were saying that there should be no *quotes* around tokens in the line (vs. having no *spaces* around the `=`).  That's why I wrote my reply like I did -- which basically completely affirmed what you said in the first place, but I didn't realize it.  😄  Sorry for the misunderstand, but we got to the same place, and the problem will be fixed shortly!
